### ISSUE TITLE
Matrix multiplication by static matrix gives scalar indexing error

### ISSUE THREAD

**Issue by jonniedie** [JuliaGPU/GPUArrays.jl#432]

**Comment by maleadt** [comment]
Yeah, sure. I wouldn't be against a PR adding the necessary functionality, but it's not very critical so I won't myself get to work on this. Realistically, you'd also want to improve the GEMM implementation here, because the fallback is awfully slow (ideally making use of GEMMKernels.jl).

**Comment by kshyatt** [comment]
@christiangnrd would you be interested in picking this up?
